docs/src/main/sphinx/installation/plugins.md

```diff
@@ -0,0 +1,311 @@
+following identifier section:
+
+```xml
+<parent>
+    <groupId>io.trino</groupId>
+    <artifactId>trino-root</artifactId>
+    <version>470</version>
+    <relativePath>../../pom.xml</relativePath>
+</parent>
+
+<artifactId>trino-postgresql</artifactId>
+<packaging>trino-plugin</packaging>
+```
+
+The Maven coordinates are therefore `io.trino:trino-postgresql:470` with version
+or `io.trino:trino-postgresql` without version. Use this term for a [search to
+locate the
+artifact](https://central.sonatype.com/search?q=io.trino%3Atrino-postgresql).
+
+The coordinates translate into a path to the ZIP archive on the Maven Central
+Repository. Use this URL to download the plugin.
+
+```
+https://repo1.maven.org/maven2/io/trino/trino-postgresql/470/trino-postgresql-470.zip
+```
+
+Availability of plugins from other projects and organizations varies widely, and
+may require building a plugin from source.
+
+When downloading a plugin you must ensure to download a version of the plugin
+that is compatible with your Trino installation. Full compatibility is only
+guaranteed when using the same Trino version used for the plugin build and the
+deployment, and therefore using the same version is recommended. Use the
+documentation or the source code of the specific plugin to confirm and refer to
+the [SPI compatibility notes](spi-compatibility) for further technical details.
+
+(plugins-installation)=
+## Installation
+
+To install a plugin, extract the ZIP archive into a directory in the `plugin`
+directory of your Trino installation on all nodes of the cluster. The directory
+contains all necessary resources. 
+
+For example, for a plugin called `example-plugin` with a version of `1.0`,
+extract the `example-plugin-1.0.zip` archive. Rename the resulting directory
+`example-plugin-1.0` to `example-plugin` and copy it into the `plugin` directory
+of your Trino installation on all workers and the coordinator of the cluster.
+
+:::{note}
+Every Trino plugin must be in a separate directory. Do not put JAR
+files directly into the `plugin` directory. Each plugin directory can only
+contain JAR files. Any subdirectories and other files are ignored. 
+:::
+
+By default, the plugin directory is the `plugin` directory relative to the
+directory in which Trino is installed, but it is configurable using the
+configuration variable `plugin.dir` with the launcher. The [Docker
+container](/installation/containers) uses the path `/usr/lib/trino/plugin`.
+
+Restart Trino to use the plugin. 
+
+(plugins-removal)=
+## Removal
+
+Plugins can be safely removed if the functionality is not needed or desired on
+your Trino cluster. Use the following steps for a safe removal across the
+cluster:
+
+* Shut down all nodes.
+* Delete the directory in the `plugin` folder of the Trino installation on all
+  nodes. 
+* Start up all nodes.
+
+More advanced deployments can also use rolling deletion and restarts of cluster
```

Comment:
Let's remove this. The plugin would need to be removed from the coordinator first, which requires restarting the coordinator, so you might as well restart the entire cluster at once.

Reply:
ok .. heads up @nineinchnick